### PR TITLE
UI Cleanup

### PR DESCRIPTION
**Pull request by jeznorth**
## Links to Jira Tickets

- [SIMSBIOHUB-4789](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-479)

## Description of Changes

- General UI clean-up and accessibility improvements

### CODE REVIEW

**Review by curtisupshall — DISMISSED** [review]
LGTM

File: app/src/features/surveys/SurveyPermitForm.tsx [hunk 1]
```diff
@@ -77,97 +64,109 @@ export const SurveyPermitFormYupSchema = yup.object().shape({
...
-                              </MenuItem>
-                            </Select>
-                            <FormHelperText>{permitTypeMeta.touched && permitTypeMeta.error}</FormHelperText>
-                          </FormControl>
-                        </Grid>
-                      </Grid>
-                    </Box>
-                  </ListItemText>
-                  <ListItemSecondaryAction>
+                <Collapse role="listitem" key={index}>
```
Comment (MacDeluca) [inline app/src/features/surveys/SurveyPermitForm.tsx:88]: Should update this key to a non index value.
https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

File: app/src/features/surveys/components/SurveyFundingSourceForm.tsx [hunk 2]
```diff
@@ -73,59 +77,82 @@ const SurveyFundingSourceForm = () => {
...
+              gap={1}
+              role="list"
+              sx={{
+                '&:not(:has(div[role=listitem]))': {
+                  display: 'none'
+                }
+              }}>
+              {values.funding_sources.map((surveyFundingSource: ISurveyFundingSource, index: number) => {
+                return (
+                  <Collapse role="listitem" key={index}>
```
Comment (MacDeluca) [inline app/src/features/surveys/components/SurveyFundingSourceForm.tsx:92]: Same here.
https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66

File: app/src/components/fields/CustomTextField.tsx [hunk 3]
```diff
@@ -14,6 +14,8 @@ export interface ICustomTextField {
   handleBlur?: FormikContextType<any>['handleBlur'];
 
   maxLength?: number;
+
+  endAdornment?: any;
```
Comment (MacDeluca) [inline app/src/components/fields/CustomTextField.tsx]: Is there a more specific type we can use for this?

File: app/src/features/surveys/SurveyPermitForm.tsx (hunk 1 above)
Comment (jeznorth) [reply app/src/features/surveys/SurveyPermitForm.tsx:88]: Ideal yes - for now we'll just leave it as mark for tech debt.

File: app/src/features/surveys/components/SurveyFundingSourceForm.tsx (hunk 2 above)
Comment (jeznorth) [reply app/src/features/surveys/components/SurveyFundingSourceForm.tsx:92]: See previous comment.

File: app/src/components/fields/CustomTextField.tsx (hunk 3 above)
Comment (jeznorth) [reply app/src/components/fields/CustomTextField.tsx]: Reverting this. Unintended change.

**Review by curtisupshall — DISMISSED** [review]

**Comment by NickPhura** [comment]
```
Failed to compile.

Error: Parse Error: <meta name="viewport" content="viewport-fit=cover, width=d evice-width, initial-scale=1.0, minimum-scale=1.0 />

<meta name=" format-detection" content="telephone=no" />
<meta name="msapplication-tap-highlight" content="no" />
<link rel="manifest" href="/manifest.json" />
<link rel="shortcut icon" type="image/png" href="/assets/icon/favicon.ico" / >
<!-- add to homescreen for ios -->
<meta name="apple-mobile-web-app-capable" content="yes" />
<meta name="apple-mobile-web-app-title" content="Species Inventory Managemen t System" />
<meta name="apple-mobile-web-app-status-bar-style" content="black" />
<script defer src="/static/js/main.90fb5321.js"></script><link href="/static/c ss/main.208c3532.css" rel="stylesheet"></head>
<body>
<div id="root"></div>
</body>
</html>

- htmlparser.js:255 HTMLParser.parse
[src]/[html-minifier-terser]/src/htmlparser.js:255:15

- runMicrotasks

- task_queues.js:95 processTicksAndRejections
internal/process/task_queues.js:95:5

- htmlminifier.js:1282 async minify
[src]/[html-minifier-terser]/src/htmlminifier.js:1282:3

- htmlminifier.js:1365 async Object.exports.minify
[src]/[html-minifier-terser]/src/htmlminifier.js:1365:16
```

**Review by MacDeluca — APPROVED** [review]
Ohhhkehhhlesssgo

**Review by curtisupshall — APPROVED** [review]

**Pull request merged**